packetfu.gemspec

```diff
@@ -2,14 +2,14 @@ require 'rake'
 require './lib/packetfu/version'
 
 Gem::Specification.new do |s|
-  s.name        = 'packetfu'
+  s.name        = 'packetfu6'
   s.version     = PacketFu::VERSION
-  s.authors     = ['Tod Beardsley', 'Jonathan Claudius']
-  s.email       = ['todb@packetfu.com', 'claudijd@yahoo.com']
-  s.summary     = 'PacketFu is a mid-level packet manipulation library.'
-  s.homepage    = 'https://github.com/packetfu/packetfu'
+  s.authors     = ['Dimitris Grigoriou']
```

Comment:
We should acknowledge the original authors by also including them here in addition to you. Similarly for the email address.